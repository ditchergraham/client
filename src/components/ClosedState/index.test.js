import { h } from 'preact';
import render from 'preact-render-to-string';

import ClosedState from '.';
import { ThemeProvider } from '../ThemeProvider';

describe('<ClosedState />', () => {
  it('matches snapshot', () => {
    const theme = {
      primary_colour: 'test',
    };
    const tree = render(<ThemeProvider theme={theme} style="messenger">
      <ClosedState toggleChat={jest.fn()} />
    </ThemeProvider>);
    expect(tree).toMatchSnapshot();
  });
});
