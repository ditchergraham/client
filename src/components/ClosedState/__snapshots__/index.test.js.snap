// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClosedState /> matches snapshot 1`] = `"<header style=\\"background-color: test;\\" class=\\"Header--messenger\\"><span class=\\"Header__title--messenger\\">Chat with John</span><button class=\\"Header__closeBtn--messenger\\"><span>...</span></button></header>"`;
